docs/rulebook.md

```diff
@@ -0,0 +1,100 @@
+# Rulebook / constitution
+
+This is a small document that defines how this community is working.
+
+## Project role
+
+Provide a shared development environment and infrastructure for the Nix and
+NixOS ecosystem, strictly for non-profit purposes.
+
+## Members
+
+There are currently two classes of members in the project:
+
+* Community members - participants of our community.
+* Administrators - members who own and administer the project.
+
+## Community members
+
+### Team
+
+Anybody who contributes code to one of our project is considered a community
+member.
+
+### Responsibilities
+
+Community members are asked to help keep a good spirit focused on building and
+shipping software. Everybody is human and shall be treated with compassion and
+respect.
+
+### Power
+
+> FIXME: this is not available yet.
+
+Community members might create a referendum. Any referendum that has more than
+10 votes gets submitted to the community at large for voting. The voting
+result will be enacted on a majority vote.
+
+## Administrators
+
+### Team
+
+* @Mic92
+* @adisbladis
+* @ryantm
+* @zimbatm
+* @zowoq
+
+### Role
+
+The role of the administrators is to support the members of the org, and the
+Nix project in general.
+
+### Responsibilities
+
+On top of the community members responsibilities, the administrators are also
+responsible for serving the community to the best of their ability and
+availability.
+
+They provide services such as:
+* keep the systems updated and patched
+* reply to user requests
+* host new services that could help the project
+* on-board new projects
+* resolve conflict
+
+### Quorum of 5 rule
+
+To reduce the attack surface on the project, the administrative team size is
+limited to 5 people.
+
+### Power
+
+The administrators have the power to change any aspect of the project.
+
+Controversial changes require a majority vote. Any administrator can call a
+change controversial :-)
```

Comment:
TODO: Either change the term "controversial" or clarify that "controversial" is not a negative term in this context. It just means that opinions differ and we need a bit more time to reach a consensus.